Security/Voter/IsOwnerVoter.php

```diff
@@ -20,13 +20,7 @@ public function supportsAttribute($attribute)
 
     public function supportsClass($class)
     {
-        if (is_object($class)) {
-            $refl = new \ReflectionObject($class);
-
-            return $refl->implementsInterface('Knp\RadBundle\Security\OwnableInterface');
-        }
-
-        return false;
+        return is_subclass_of($class, 'Knp\RadBundle\Security\OwnableInterface');
```

Comment:
The new method will accept a string as value (like a class name implementing `OwnableInterface`, whereas the old one did not.